build.sbt

```diff
@@ -1,10 +1,10 @@
 ThisBuild / tlBaseVersion := "2.11"
 
-val scalaCheckVersion = "1.17.1"
+val scalaCheckVersion = "1.18.0"
 
-val disciplineVersion = "1.6.0"
+val disciplineVersion = "1.7.0"
 
-val disciplineMunitVersion = "2.0.0-M3"
+val disciplineMunitVersion = "2.0.0-M4"
```

Comment:
That's right. However, I think it wouldn't hurt if we address it in a separate PR to avoid stuffing up this one.

It would be perfect, if @typelevel-steward could wake up and create such a PR for us.